ci/build.yml

```diff
@@ -249,6 +249,7 @@ jobs:
         ARTIFACTORY_USERNAME: $(ARTIFACTORY_USERNAME)
         ARTIFACTORY_PASSWORD: $(ARTIFACTORY_PASSWORD)
       displayName: 'Build'
+      condition: eq(variables['System.PullRequest.IsFork'], 'False')
```

Comment:
I’m sure you’ve checked it but I probably wouldn’t have so let me ask the silly question anyway: Is this set on `main`?

Reply:
Yes, it's a [predefined variable](https://docs.microsoft.com/en-us/azure/devops/pipelines/build/variables?view=azure-devops&tabs=yaml) and is always set to either `"False"` or `"True"`.